ui/guest/src/components/DropMarker.tsx

```diff
@@ -17,16 +17,32 @@
 
 import React, { useEffect, useRef, useState } from 'react';
 import {
-  findClosestRect, foo,
+  findClosestRect,
+  foo,
+  forEach,
   getDropMarkerPosition,
   getInRectStats,
   HORIZONTAL,
   notNullOrUndefined,
+  splitRect,
   VERTICAL,
-  forEach, splitRect, X_AXIS, Y_AXIS
+  X_AXIS,
+  Y_AXIS
 } from '../util';
 
-export function DropMarker(props) {
+interface DropMarkerProps {
+  over: any;
```

Comment:
You may debug things in the browser or review who uses a given artifact to help you identify types you're unclear.

Reply:
To date, this is from 11 days ago and still pending.